registry/app.go

```diff
@@ -88,6 +88,8 @@ func NewApp(configuration configuration.Configuration) *App {
 }
 
 func (app *App) ServeHTTP(w http.ResponseWriter, r *http.Request) {
+	// Set a header with the Docker Distribution API Version for all responses.
+	w.Header().Set("Docker-Distribution-API-Version", "2.0")
```

Comment:
@stevvooe Sounds good, but the last example could be done with multiple instances of the header instead. I'll update.